Rubberduck.Refactorings/EncapsulateField/FieldCandidates/EncapsulateFieldCandidate.cs

```diff
@@ -47,11 +45,13 @@ public class EncapsulateFieldCandidate : IEncapsulateFieldCandidate
         protected int _hashCode;
         private string _identifierName;
         protected EncapsulationIdentifiers _fieldAndProperty;
+        private Func<string, string> _paramNameBuilder;
 
-        public EncapsulateFieldCandidate(Declaration declaration, IValidateVBAIdentifiers identifierValidator)
+        public EncapsulateFieldCandidate(Declaration declaration, IValidateVBAIdentifiers identifierValidator, Func<string, string> paramNameBuilder)
```

Comment:
I am not a fan of shortening parameter and variable names.
However, here it is good-enough as it is.